dspace/config/dspace.cfg

```diff
@@ -1455,34 +1455,15 @@ sitemap.engineurls = http://www.google.com/webmasters/sitemaps/ping?sitemap=
 #
 # No known Sitemap 'ping' URL for MSN/Live search
 
-#####  SHERPA/Romeo Integration Settings ####
-# the SHERPA/RoMEO endpoint
-sherpa.romeo.url = http://www.sherpa.ac.uk/romeo/api29.php
```

Comment:
I don't think we need to remove these configurations from our dspace.cfg.  Please remember, that in Spring Bean definitions we can *reference* these configurations using `${config}` references.  Please take a look at how we do this in our `core-hibernate.xml` to pull in the database settings from dspace.cfg: https://github.com/DSpace/DSpace/blob/master/dspace/config/spring/api/core-hibernate.xml#L41-L49

I'd rather make these configurations EASY to configure and override (in your local.cfg).  So I think we need to keep these configurations in dspace.cfg instead of moving them into Spring XML configs (which are more difficult to edit & override).

Reply:
would like to participate in a more general discussion, elsewhere, of how we configure DSpace.

While @tdonohue 's concerns are reasonable, I wonder how likely it is that this will ever be touched, unless the remote service changes (in which case, we will *all* have to touch it).  There is something to be said for making things that normally shouldn't be adjusted, hard to mess up.

Reply:
@mwoodiupui : I agree with your logic overall, but I think my original comment was taken slightly out of context.  You are correct, we may want to argue that the SHERPA API *url* is unlikely to need overriding. However, there's also a SHERPA "apikey" that will *always need overriding* but is unfortunately buried in Spring configs here: https://github.com/DSpace/DSpace/pull/2560/files#diff-c1d29d61b7b458582c79b2ac9383b356R13

So, my comment above was more to say "Let's please not move configs (in general) into Spring XML files, when we can just as easily reference them where they exist in `dspace.cfg`"  I wasn't meaning to imply this only applies to the URL setting.

That said, honestly, we might also want to keep the `sherpa.romeo.url` in the dspace.cfg for now, simply because it's used *twice* in Spring configs.  It'd be much easier to change in one place (when necessary).

Reply:
Ah, yes, API keys are site configuration, not developer configuration, and should be readily accessible by site maintainers.  And the corresponding URL may as well stay with it for now.